README.md

```diff
@@ -118,6 +118,9 @@ Here are all of the extension settings with their default values. To change any
   // Use garavatar icons in log viewers
   "svn.gravatars.enabled": true,
 
+  // Set up client-side hooks to be executed before or after SVN commands.
+  "svn.hooks": null
```

Comment:
This should just be an empty array otherwise you will need to add null checks

Reply:
Good catch! I'll fix this soon along with other issues I have identified.